tests/unit_test_external.py

```diff
@@ -1394,35 +1271,30 @@ def test_constant_v3d(self):
         tab = "Vertical"
         cell = "C5*"
         cell2 = "F5*"
-        coords = {'XY': ['X'],
-                  'ABC': ['A', 'B', 'C'],
+        coords = {'ABC': ['A', 'B', 'C'],
```

Comment:
is this because dictionaries are ordered now?

Reply:
Yes, the previous version of the test was taken the order from te dims list. However, now dims argument has been removed from the external objects, as the coords can manage all the information (coordinates and order of the dimensions). This forced to redefine the coords dictionaries in the test to be with the same order as the dims list was.

Reply:
that makes sense. thanks!